gslb/ingestion/gslb.go

```diff
@@ -854,7 +864,9 @@ func InitializeGSLBMemberClusters(membersKubeConfig string, memberClusters []gsl
 		}
 		aviCtrl, err := InitializeMemberCluster(cfg, cluster, clients)
 		if err != nil {
-			return nil, fmt.Errorf("error initializing member cluster %s: %s", cluster.clusterName, err)
+			gslbutils.Warnf("error initializing member cluster %s: %s", cluster.clusterName, err)
+			pendingClusters[cluster] = true
```

Comment:
Adding `true` here seems like we are checking this value else where, which we aren't. Do you think it would be better to just set an empty struct here?

Reply:
Yeah makes sense. Updated